sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -117,7 +118,7 @@ class RadarLocationReceiver : BroadcastReceiver() {
             }
             ACTION_LOCATION -> {
                 val result = LocationResult.extractResult(intent)
-                result?.lastLocation?.also {
+                result.lastLocation.also {
```

Comment:
Same thing here, seems like `result` and `lastLocation` could be null:

https://developers.google.com/android/reference/com/google/android/gms/location/LocationResult#extractResult(android.content.Intent)

Reply:
same here

Reply:
Looks like these are driven through _contracts_, such that if the given `intent` is null, then they will return null. But, since the intent is NonNull, then no null check is needed.

Reply:
The decompiled source code for this looked like it could still return null - but the above one looks ok.